astropy/coordinates/representation.py

```diff
@@ -223,6 +224,27 @@ def _unitstr(self):
                            for component in self.components]))
         return unitstr
 
+    def __eq__(self, other):
+        """
+        Check for equality with another representation.
+        """
+        try:
+            comparisons = [getattr(self, component) ==
+                           getattr(other, component)
+                           for component in self.components]
+        except:
```

Comment:
bare except seems wrong. I think it should at least be `Exception`. The same applies to the except 5 lines later.